mw-webapp/src/component/table/Table.tsx

```diff
@@ -4,7 +4,7 @@ import styles from "src/component/table/Table.module.scss";
 /**
  * Tables data
  */
-interface TableData<T> {
+export interface TableData<T> {
```

Comment:
why we need export here? Do you use this props in other files too?

Reply:
Forgot to change, done